GraphEditor/GraphEditor.cpp

```diff
@@ -554,13 +555,9 @@ void GraphEditor::handleCopy(void)
         jsonObjs.push_back(obj->serialize());
     }
 
-    //to byte array
-    const QJsonDocument jsonDoc(jsonObjs);
-    const auto data = jsonDoc.toBinaryData();
-
     //load the clipboard
     auto mimeData = new QMimeData();
-    mimeData->setData("binary/json/pothos_object_array", data);
+    mimeData->setData("binary/json/pothos_object_array", QJsonDocument::fromVariant(jsonObjs).toJson());
```

Comment:
I think it would be better if this was replaced with [QCborValue/QCborArray/QCborMap](https://doc.qt.io/qt-5/qcborvalue.html), but need input from others.